kafka_consumer/tests/test_unit.py

```diff
@@ -32,24 +30,6 @@ def test_tls_config_legacy(extra_config, expected_http_kwargs, check, kafka_inst
     assert expected_http_kwargs == actual_options
 
 
-def test_invalid_connect_str(dd_run_check, check, aggregator, caplog, kafka_instance):
-    caplog.set_level(logging.DEBUG)
-    kafka_instance['kafka_connect_str'] = 'invalid'
-    del kafka_instance['consumer_groups']
-    dd_run_check(check(kafka_instance))
-
-    for m in metrics:
-        aggregator.assert_metric(m, count=0)
-
-    exception_msg = (
-        'ConfigurationError: Cannot fetch consumer offsets because no consumer_groups are specified and '
-        'monitor_unlisted_consumer_groups is False'
-    )
-
-    assert exception_msg in caplog.text
-    aggregator.assert_metrics_using_metadata(get_metadata_metrics())
-
-
```

Comment:
Note on why I removed this:

The purpose of this test was unclear and had a misleading name, and this is being tested in other tests anyway, so it seems like removing is the best course of action (tests that we don't understand can be a liability more than an asset).